react/features/speaker-stats/components/SpeakerStatsSearch.js

```diff
@@ -0,0 +1,74 @@
+/* @flow */
+
+import { FieldTextStateless as TextField } from '@atlaskit/field-text';
+import { makeStyles } from '@material-ui/core/styles';
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { getFieldValue } from '../../base/react';
+
+const useStyles = makeStyles(() => {
+    return {
+        speakerStatsSearch: {
+            position: 'absolute',
+            right: '50px',
+            top: '-5px'
+        }
+    };
+});
+
+/**
+ * The type of the React {@code Component} props of {@link SpeakerStatsSearch}.
+ */
+type Props = {
+
+    /**
+     * The function to initiate the change in the speaker stats table.
+     */
+    onSearch: Function,
+
+};
+
+/**
+ * React component for display an individual user's speaker stats.
+ *
+ * @returns {React$Element<any>}
+ */
+function SpeakerStatsSearch({ onSearch }: Props) {
+    const classes = useStyles();
+    const { t } = useTranslation();
+    const [ searchValue, setSearchValue ] = useState<string>('');
+    const onChange = useCallback((evt: Event) => {
+        const value = getFieldValue(evt);
+
+        setSearchValue(value);
+
+        onSearch && onSearch(value);
+    }, []);
+    const disableSpeakerStatsSearch = useSelector(
+        (state: Object) => state['features/base/config']?.disableSpeakerStatsSearch
```

Comment:
Please extract the selector in a meaningfully named function in a separate file, i.e. `react > features > speaker-stats > functions.js`

Reply:
Done